covid_graphs/covid_graphs/formula.py

```diff
@@ -117,20 +117,23 @@ def get_trace_generator(self, country_report: CountryReport) -> TraceGenerator:
         # The choice of date zero is in theory arbitrary.
         date_zero = country_report.dates[0]
         xs = [(date - date_zero).days for date in country_report.dates[: until_idx + 1]]
-        fit = fit_atg_model.fit_atg_model(
+        self.fit = fit_atg_model.fit_atg_model(
```

Comment:
I've changed `fit` into `self.fit`. This is later accessed in `store_predictions`. This is ugly, but I'm okay with taking some technical debt for now.

If you have simple suggestions to make this better, I can incorporate them now. Otherwise it will have to wait after our release.

Reply:
Okay, went with the constructor. It was possible at both call-sites.